UI/CMakeLists.txt

```diff
@@ -26,6 +26,12 @@ if(TARGET obs-browser
     obs-browser-panels INTERFACE ${CMAKE_SOURCE_DIR}/plugins/obs-browser/panel)
 endif()
 
+set(OAUTH_BASE_URL
```

Comment:
Why use a cache variable here? If it's not supposed to be changeable by the user (which it isn't, you `force` the value over whatever a user specifies), make it either a normal project variable (by using `set` without qualifiers) or make it an internal cache variable (which usually only makes sense if you need it globally).

Reply:
This was essentially just copied over from https://github.com/obsproject/obs-studio/blob/master/plugins/rtmp-services/CMakeLists.txt#L5-L9

The variable is supposed to be changeable in some cases (e.g. Linux packages or developers running their own OAuth servers since they don't have access to our credentials).

So in order to achieve that would I need to simply remove `FORCE` or also another part of it?

Reply:
Yeah remove the `FORCE` part or only set+force it if the value is an empty or illegal string.